contracts/hydro/src/msg.rs

```diff
@@ -44,6 +47,7 @@ pub enum ExecuteMsg {
     },
     UnlockTokens {},
     CreateProposal {
+        round_id: Option<u64>,
```

Comment:
Not sure if new proposals are created through some 3rd party tool/app and if this possibly matters.

Reply:
It is done via Celatone, so no worries here. Just needs a schema update

Reply:
Ok, schema is updated, we are good there.